mobilecoind/api/proto/mobilecoind_api.proto

```diff
@@ -443,6 +445,24 @@ message GetBlockInfoResponse {
     uint64 txo_count = 2;
 }
 
+// Get more detailed information about a downloaded block
+message GetBlockRequest {
+    uint64 block = 1;
+}
+message GetBlockResponse {
+    // Key images in the block
+    repeated external.KeyImage key_images = 1;
+
+    // TxOuts in the block.
+    repeated external.TxOut txos = 2;
+
+    // The block signature
+    blockchain.BlockSignature signature = 3;
+
+    // The block
+    blockchain.Block block = 4;
```

Comment:
I would prefer to have these things ordered in a more logical way - the block, the signature, key images, and then txos.